CHANGELOG.md

```diff
@@ -4,6 +4,20 @@ This file contains notable changes (e.g. breaking changes, major changes, etc.)
 
 This file was introduced starting Kani 0.23.0, so it only contains changes from version 0.23.0 onwards.
 
+## [0.43.0]
+
+## What's Changed
+* Make `codegen_span` stable by @ouz-a in https://github.com/model-checking/kani/pull/2882
```

Comment:
Unfortunately I don't think any of these changes are really customer visible. If you want, I would just merge all of these into one and list the contributors

Reply:
I just removed all of them.